config.ru

```diff
@@ -1,6 +1,8 @@
+require 'newrelic_rpm'
 require 'cellect/server'
 require_relative 'lib/cellect/server/adapters/panoptes'
 
+p ENV['RAILS_ENV']
```

Comment:
My bad I removed that I was trying to debug why newrelic wasn't reporting to it's server.